app/models/generic_event/journey_arrive_at_outer_gate.rb

```diff
@@ -1,5 +1,7 @@
 class GenericEvent
   class JourneyArriveAtOuterGate < GenericEvent
+    DETAILS_ATTRIBUTES = %w[].freeze
```

Comment:
To avoid having empty constants in classes that don't support details, is it possible to default to no detail attributes unless they exist in a class, or maybe move the empty `DETAILS_ATTRIBUTES` to the generic event base class?

Reply:
So I'm not sure. It requires a small amount of maintenance for more visibility of the possible details when you look at the individual event. I could/did swing either way on this. If you feel strongly I can DRY it up a bit.